roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -75,21 +77,19 @@ data:
 
             root /usr/share/nginx/html;
 
-            location ~ ^/api/eda/v[0-9]+/ {
+            location ~ ^/api/eda {
                 proxy_pass http://{{ api_server_name }}:{{ api_nginx_port }};
                 proxy_set_header Origin http://{{ api_server_name }}:{{ api_nginx_port }};
                 proxy_set_header Host $http_host;
-                proxy_set_header X-Forwarded-Host $host;
                 proxy_set_header X-Forwarded-Proto $remote_scheme;
                 proxy_set_header X-Forwarded-Port $server_port;
                 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             }
 
-            location ~ ^/static/ {
+            location ~ ^/api/eda/static/ {
```

Comment:
you don't need this location, is mostly duplicated with the previous one. `location ~ ^/api/eda`